pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -714,18 +853,19 @@ func getGroup(spec *specs.Spec, groupPath string, filter func(user.Group) bool)
 	return groups[0], nil
 }
 
-func (policy *regoEnforcer) GetUserInfo(spec *oci.Spec) (IDName, []IDName, string, error) {
-	passwdPath := filepath.Join(spec.Root.Path, "/etc/passwd")
-	groupPath := filepath.Join(spec.Root.Path, "/etc/group")
+func (policy *regoEnforcer) GetUserInfo(containerID string, process *oci.Process) (IDName, []IDName, string, error) {
+	rootPath := filepath.Join(guestpath.LCOWRootPrefixInUVM, containerID, guestpath.RootfsPath)
```

Comment:
as discussed offline, this is probably fine for now, but we need to make sure that we're using the actual value of the root, since the host can send a different value at container create time.

Reply:
huh?

Reply:
it probably was myself and @matajoh